src/browser/ScriptManager.zig

```diff
@@ -801,7 +801,11 @@ pub const Script = struct {
             log.debug(.browser, "executed script", .{ .src = url, .success = success, .on_load = script_element._on_load != null });
         }
 
-        defer page.tick();
```

Comment:
This'll run microtasks, scheduled tasks, and microtasks again, is that right?

Maybe it should just be:

```zig
    _ = self.scheduler.run() catch |err| {
        log.err(.page, "scheduler", .{ .err = err });
    };
    self.js.runMicrotasks();
```

and remove page.tick.